---
title: Phylogenetic analysis of COVID-19 spread in Australia and New Zealand.
authors:
  - Dr Pleayo Tovaranonte
authorLinks:
  - https://www.linkedin.com/in/pleayo/
affiliations: "Precision Medicine International Ltd, Auckland, New Zealand"
translators:
translatorLinks:
license: "CC-BY"  
licenseLink: "https://creativecommons.org/licenses/by/4.0/"
dataset: "https://nextstrain.org/ncov/oceania?c=location&dmin=2019-12-25"

abstract: "This report uses publicly shared phylogenetic data to track the spread of COVID-19 in Australia and New Zealand. We analyse the genetic mutations of the SARS-CoV-2 virus and attempt to explain how virus subtypes affect epidemiology and severity of disease."
---
<!-- Translators: Only text after : in the above ^ needs to be translated -->
<!-- Comment tags like these do not need to be translated, they are only to help you! -->
<!-- Ensure that links always end in a 'letter' (. counts) If some kind of text doesn't follow them, it breaks the slide. -->
<!-- numbers can be tagged ilke this: 161</tag> - this is just for us to help find them to update! Just leave in the </tag> bit. -->

<!-- This is left-side text 1-->
# [Table of Contents](https://nextstrain.org/community/narratives/pleayo/covid/AusNZ?d=tree,map&p=grid)

* [Background resources](https://nextstrain.org/community/narratives/pleayo/covid/AusNZ?n=2).     
* [A note of sampling](https://nextstrain.org/community/narratives/pleayo/covid/AusNZ?n=3).  
* [Origins of the epidemic](https://nextstrain.org/community/narratives/pleayo/covid/AusNZ?n=4).  
* [COVID-19 cases in Australia](https://nextstrain.org/community/narratives/pleayo/covid/AusNZ?n=5).  
* [COVID-19 cases in New Zealand](https://nextstrain.org/community/narratives/pleayo/covid/AusNZ?n=6).  
* [Regional spread across South-East Asia](https://nextstrain.org/community/narratives/pleayo/covid/AusNZ?n=7).    
* [Takeaways](https://nextstrain.org/community/narratives/pleayo/covid/AusNZ?n=8).
* [Scientific credit](https://nextstrain.org/community/narratives/pleayo/covid/AusNZ?n=9).

<!-- This is right-side text -->
```auspiceMainDisplayMarkdown
# Executive summary

We analyzed over 2,000 publicly shared SARS-CoV-2 genomes through the community platform, Nextstrain. By comparing the genetic compositions of the SARS-CoV-2 virus based on their similarities and chronological timeline, we can characterise how the COVID-19 spreads around the world and focus on Australia and New Zealand. In particular, we look at:


* Sampling methods  
* Origins of the epidemic
* Phylogentic analysis of COVID-19 cases in Australia
* Phylogentic analysis of COVID-19 cases in New Zealand
* Regional spread across South-East Asia
```


<!-- ############ SLIDE BREAK ############# -->

<!-- This is left-side text 2-->
# [COVID-19 Resources](https://nextstrain.org/ncov/asia/2020-05-08?d=tree&p=full&legend=closed&f_region=Asia)
Some resources that will make interpreting the data we present in this narrative easier.
#### Nextstrain Resources  
* [START HERE: How to read a phylogeny](https://nextstrain.org/narratives/trees-background/).  
* [Background on coronaviruses](https://nextstrain.org/help/coronavirus/human-CoV).

#### External Resources  
* [WHO Situation Reports](https://www.who.int/emergencies/diseases/novel-coronavirus-2019/situation-reports).
* [CDC Resources](https://www.cdc.gov/coronavirus/2019-ncov/index.html).
* [Department of Health, Australia](https://www.health.gov.au/news/health-alerts/novel-coronavirus-2019-ncov-health-alert/coronavirus-covid-19-current-situation-and-case-numbers).
* [Ministry of Health, New Zealand](https://www.health.govt.nz/our-work/diseases-and-conditions/covid-19-novel-coronavirus/covid-19-current-situation/covid-19-current-cases).
<!-- There is no right-side text -->


<!-- ############ SLIDE BREAK ############# -->

<!-- This is left-side text -->
# [Sampling Methods](https://nextstrain.org/ncov/oceania?c=location&dmin=2019-12-25)
We currently have sequences from samples taken in 2 countries - Australia and New Zealand. This is a collaborative achievement between the trans Tasman partnership. Sequencing an unknown, large RNA virus in the midst of a pandemic is difficult, and is only possible through the incredible work and timely sharing of data by scientists and physicians around the world.
<br><br>
The most common screening test is the detection of the E-gene by real-time PCR, followed by a confirmatory test by the detection of the RdRP-gene by real-time PCR. Other tests available include N-gene, Orf1b-gene, virus isolation, and whole genome sequencing.
<br><br>
The most preferred sampling method is from the lower respiratory tract (sputum, endotrachial excretions, or brocho-alveolar lavage).
However, the most commonly performed sampling method is nasopharyngal swabbing. An alternative method is, of course, a post-mortem lung tissue biopsy.
<br><br>
While this data enables us to infer many useful characteristics of the outbreak and track its spread in real time, it's important to emphasize that our conclusions are limited by the available data.
<br><br>
For example, the map shows very few sequences from the Pacific Islands. This is NOT because COVID-19 is not circulating in these areas, or that these cases are not as crucial to understand; rather, we do not have sufficient data available from these areas.

<!-- There is NO right-side text -->


<!-- ############ SLIDE BREAK ############# -->
<!-- This is left-side text -->
# [Origins of the epidemic](https://nextstrain.org/ncov/oceania?c=location&dmin=2019-12-25)

This phylogeny shows evolutionary relationships of hCoV-19 (or SARS-CoV-2) viruses from the ongoing novel coronavirus COVID-19 pandemic. This phylogeny shows an initial emergence in Wuhan, China, in Nov-Dec 2019 followed by sustained human-to-human transmission leading to sampled infections. Although the genetic relationships among sampled viruses are quite clear, there is considerable uncertainty surrounding estimates of specific transmission dates and in reconstruction of geographic spread. Please be aware that specific inferred transmission patterns are only a hypothesis.
<br><br>
There are thousands of complete genomes available now and this number increases by hundreds of every day. This visualization can only handle ~3000 genomes in a single view for performance and legibility reasons. Because of this we have to subsample available genome data for these analysis views. Our primary global analysis subsamples to 120 genomes per admin division per month. This will result in a more equitable global sequence distribution, but it hides samples available from regions that are doing lots of sequencing.
<br><br>
Site numbering and genome structure uses Wuhan-Hu-1/2019 as reference. The phylogeny is rooted relative to early samples from Wuhan. Temporal resolution assumes a nucleotide substitution rate of 8 Ã— 10^-4 subs per site per year.
<!-- There is no right side text -->


<!-- ############ SLIDE BREAK ############# -->
<!-- This is left-side text -->
# [Phylogentic analysis of COVID-19 cases in Australia](https://nextstrain.org/ncov/oceania?f_country=Australia)

In the last 3 months, SARS-CoV-2 has been introduced to Australia many times, from multiple locations.
<br><br>
Most of these introductions have led to limited onward transmission (single cases or small clusters on the tree).
<br><br>
In the middle of the tree, however, we see a single introduction that led to extensive onward transmission.
This large, cohesive cluster is most closely related to samples from China and Italy.
<br><br>

<br><br>
This illustrates how the extent to which an introduction spreads within a population depends on sociological factors and emphasizes why it is critically important to protect vulnerable groups.
<!-- There is no right side text -->


<!-- ############ SLIDE BREAK ############# -->
<!-- This is left-side text -->
# [Phylogentic analysis of COVID-19 cases in New Zealand](https://nextstrain.org/ncov/oceania?f_country=New%20Zealand)

We have a limited number of sequences from New Zealand on Nextstrain, sampled between late January and late March. These sequences group with several different clades, each of which came from Europe and later from South East Asia. This suggests that there have been multiple introductions from the surrounding region into New Zealand.


<!-- ############ SLIDE BREAK ############# -->
<!-- This is left-side text -->
# [Regional spread across South-East Asia](https://nextstrain.org/ncov/asia/2020-05-08?d=tree,map&f_country=Japan,Jordan,Thailand,United%20Arab%20Emirates)

A similar pattern is repeated for many other countries in the region: multiple introductions, many of which were well-contained, but with a few that took off to spark local transmission. We encourage you to "Explore the Data Yourself". As shown here, you will observe similar patterns in Jordan, Japan, Thailand, and the United Arab Emirates. Similar patterns can also be seen for Kuwait, Georgia, Taiwan, and Vietnam.
<br><br>
The consistency of this pattern is strong evidence that the virus frequently crosses international borders. Thus, in order to effectively control the epidemic in any one country, we must control the pandemic across all countries through coordinated international efforts.
<!-- There is no right side text -->


<!-- ############ SLIDE BREAK ############# -->
<!-- This is left-side text 13-->
# [What you can do](https://nextstrain.org/ncov/2020-05-08?c=country&d=map&p=full)
#### ...as an individual
* Practice strict social distancing, especially if you are in a vulnerable group.
* Remember that even if you are not super vulnerable, many people around you are; follow these practices to protect others.
* Wash your hands "like you just chopped a jalapeno and have to change a contact lens."  
* Stay home as much as possible -- especially  if you are sick; be prepared with extra supplies in case you need to self-quarantine.  
* If you are an employer, encourage your employees to work from home wherever possible.

#### ...as an official  
* Make testing free and broadly available.  
* Put strong social distancing measures in place.  
* Fund and implement extensive contact tracing efforts.  
* Financially support those impacted by social distancing measures.


<!-- This is the right-side text -->

```auspiceMainDisplayMarkdown
# Takeaways

#### The outbreak in India is the result of many separate introductions from around the world.  
<br><br>
#### In Singapore, sociological factors accelerated the expansion of one introduction to become a large outbreak, in contrast to the many introductions which have not spread widely.
<br><br>
#### In Asia, most countries' outbreaks have a similar history: many separate introductions from other countries, some of which resulted in local spread. This pattern of multiple introductions over time emphasizes the need for international and regional cooperation to fully control the pandemic.
```

<!-- ############ SLIDE BREAK ############# -->






<!-- ############ SLIDE BREAK ############# -->

<!-- This is left-side text -->
# [Scientific credit](https://nextstrain.org/ncov/2020-05-06?d=map&c=author)

We would like to acknowledge the amazing and timely work done by all scientists involved in this outbreak.
Only through the rapid sharing of genomic data and metadata are analyses such as these possible.
<br><br>
The specific sequences provided by each lab are [listed here](https://github.com/nextstrain/ncov/commit/75edf84600fdf9ca6ef086d0aefd2978bc621eca).
<br><br>
I also gratefully acknowledge GISAID for providing the platform through which these data can be uploaded and shared.

<!-- Do not need to translate institutions names -->
<!-- This is right-side text -->
```auspiceMainDisplayMarkdown

I am grateful for the data gathered by these originating labs:

* AIDS Vaccine Research Laboratories
* ARS Algarve - Laboratorio Laura Ayres
* AZ Department of Health Services
* AZ SPHL, Arizona Department of Health Services
* Akershus University Hospital, Department for Microbiology and Infectious Disease Control
* Alaska State Virology Laboratory
* Andersen Lab, The Scripps Research Institute
* Arizona Department of Health Services
```
